### PR TITLE
Adding FontTest to test Altspace's fonts with a variety of Unicode ranges

### PR DESCRIPTION
**Pull request by patole**
Test is as simple as start and observe. All our fonts are stress-tested with Latin, Korean, Japanese, and Chinese characters, which cycle automatically every 3 seconds.

### CODE REVIEW

File: packages/functional-tests/src/tests/font-test.ts [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+			{blockStart: 0x20, blockEnd:0x07E, name:"Latin Basic"},
+			{blockStart: 0xA0, blockEnd: 0xFF, name:"Latin Suppliment"},
+			{blockStart: 0x100, blockEnd:0x017F, name:"Latin Extended - A"},
+			{blockStart: 0x3060, blockEnd:0x309F, name:"Japanese Hiragana/Katakana"},
+			{blockStart: 0x4E00, blockEnd:0x4E7F, name:"Chinese/Japanese/Korean Ideographs"});		
+
+		this.textBlocks.forEach((v,i,a) =>{ this.createDisplayString(v); });
+
+
+		const position = new Vector3(0,2.5,-.3);
```
Comment (tombuMS) [inline packages/functional-tests/src/tests/font-test.ts:78]: >Vector3 [](start = 23, length = 7)

Prefix with MRE and remove the extra include to keep it consistent with other tests

File: packages/functional-tests/src/tests/font-test.ts [hunk 2]
```diff
@@ -0,0 +1,221 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+
+import { Test } from '../test';
+import { Vector3 } from '@microsoft/mixed-reality-extension-sdk';
```
Comment (tombuMS) [inline packages/functional-tests/src/tests/font-test.ts:9]: Remove in favor of the MRE one above.

File: packages/functional-tests/src/tests/font-test.ts (hunk 1 above)
Comment (patole) [reply packages/functional-tests/src/tests/font-test.ts:78]: Yeah I need to find the setting for these auto imports, every time I try the lazy way without  the MRE. it adds the import and I forget to double check.
